src/redux/actionTypes.js

```diff
@@ -28,3 +28,10 @@ export const SUBMITTING_SOCIAL_AUTH = "SUBMITTING_SOCIAL_AUTH";
 export const SETTING_TOKEN_SOCIAL_AUTH = "SETTING_TOKEN_SOCIAL_AUTH";
 export const SOCIAL_AUTH_SUCCESS = "SOCIAL_AUTH_SUCCESS";
 export const SOCIAL_AUTH_FAILED = "SOCIAL_AUTH_FAILED";
+
+/**
+ * @description notification action types
+ */
+export const FETCHING = "FETCHING";
+export const SUCCESS = "SUCCESS";
```

Comment:
Make these action type more specific to avoid confusion. e.g FETCHING_NOTIFICATION